src/services/userService.ts

```diff
@@ -0,0 +1,42 @@
+import { getRepository, DeleteResult, Repository } from 'typeorm';
+import { User } from '../entity/user';
+import { TutorInfo } from '../entity/tutorInfo';
+
+class UserService {
+	static getAll = async (): Promise<User[]> => {
+		const userRepository: Repository<User> = getRepository(User);
+		return await userRepository.find({
+			select: ['id', 'username', 'roles'], //We dont want to send the passwords on response
```

Comment:
Skal vi også hente deres tutorinfo ud her?

Reply:
dette er generelt for user service, om vi ikke skal hente tutorinfo ud

Reply:
Se #18